LocalSettings.php

```diff
@@ -1496,6 +1497,7 @@
 		'test1wiki' => true,
 		'sidemwiki' => true,
 		'testwiki' => true,
+		'unionnorteamericanawiki' = true,
```

Comment:
same as above.

Reply:
Crap, I must have forgotten to enter it in.

Reply:
still missign the second one "=>"